control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -319,6 +319,65 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/nodes/{id}/label/{label}':
```

Comment:
```suggestion
  '/nodes/{id}/labels/{labels}':
```

Reply:
Only one label can be provided at a time , so IMO `label` is appropriate

Reply:
I misunderstood, thought we were adding labels